packages/animations/example/pubspec.yaml

```diff
@@ -5,6 +5,7 @@ version: 0.0.1
 
 environment:
   sdk: ">=2.3.0 <3.0.0"
+  flutter: ^1.15.3
```

Comment:
This will make pub mad because that version doesn't exist yet (from pub's point of view).